api/lib/infrastructure/utils/date-utils.js

```diff
@@ -1,5 +1,11 @@
 const moment = require('moment-timezone');
 
+moment.parseTwoDigitYear = function(yearString) {
+  const year = parseInt(yearString);
+  const currentYear = new Date().getFullYear();
+  return 2000 + year <= currentYear ? 2000 + year : 1900 + year;
```

Comment:
```suggestion
  return 2000 + year < currentYear ? 2000 + year : 1900 + year;
```
Je serai plutôt tentée de mettre strictement inférieur. On a plus de chances d'avoir un centenaire sur la plateforme Pix qu'un nourrisson ?

Reply:
Make sense ;)